apps/system/js/media_recording.js

```diff
@@ -250,22 +250,6 @@
       window.dispatchEvent(event);
       this.icon && this.icon.update();
     },
-
-    /**
-     * Return the formatted time string
```

Comment:
Yeah, the callbackj has been removed here: https://github.com/mozilla-b2g/gaia/commit/3a43272165d37848f7a2ac0cb1689209f604c5a6#diff-632d97502fb89a60d988d489b81d9492L89 and it seems that we just forgot to remove the function.